test/integration/counter_test.ts

```diff
@@ -19,12 +19,16 @@ describe('Counter', function () {
       root.k1.length.increase(3.5);
     });
     assert.equal(`{"k1":{"age":6,"length":13}}`, doc.toSortedJSON());
+    assert.equal(6, doc.getRoot().k1.age?.getValue());
```

Comment:
Both checking results of `toSortedJSON()` and `getRoot()` seems necessary: while `toSortedJSON()` refers `Document.root`, `getRoot` returns a proxy of `Document.clone`.

Reply:
Currently, Proxy refers `Clone` to handle transactions(to discard modifications in case of transaction failure). If we introduce `Inverse Operation` for `Undo/Redo` later, we can fix it to refer to the original(`Document.root`).